### PR TITLE
Install page ff: remove sidebarMenuItems from `install-landing.json`

### PR DESCRIPTION
**Pull request by heatlikeheatwave**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-heat-ff-choreedit-install-landing-json-hashicorp.vercel.app/) 🔎
- [Asana task](https://app.asana.com/0/1204817897905865/1206106982415298/f) 🎟️

## 🗒️ What

- Removed `sidebarMenuItems` from `install-landing.json`, except for Consul, which now has a Kubernetes Installs section with the following links:
  - https://developer.hashicorp.com/consul/docs/k8s/installation/install-cli
  - https://developer.hashicorp.com/consul/docs/k8s/installation/install 

## 🤷 Why

- Through discovery for the install page enhancement project, it was determined this content in the sidebar didn't add much value to the page. Asana comment [here](https://app.asana.com/0/0/1206106982415298/1206112785610849/f)

## 📸 Design Screenshots
consul

### Boundary
- Navigate to preview link for Boundary install [page](https://dev-portal-git-heat-ff-choreedit-install-landing-json-hashicorp.vercel.app/boundary/install)
- There should be no menu items between Next Steps and Resources

### Nomad
- Navigate to preview link for Nomad install [page](https://dev-portal-git-heat-ff-choreedit-install-landing-json-hashicorp.vercel.app/nomad/install)
- There should be no menu items between Next Steps and Resources

### Vault
- Navigate to preview link for Vault install [page](https://dev-portal-git-heat-ff-choreedit-install-landing-json-hashicorp.vercel.app/vault/install)
- There should be no menu items between Next Steps and Resources

### Waypoint
- Navigate to preview link for Waypoint install [page](https://dev-portal-git-heat-ff-choreedit-install-landing-json-hashicorp.vercel.app/waypoint/install)
- There should be no menu items between Next Steps and Resources

### Vagrant
- Navigate to preview link for Vagrant install [page](https://dev-portal-git-heat-ff-choreedit-install-landing-json-hashicorp.vercel.app/vagrant/install)
- There should be no menu items between the OS menu items and Resources

### Vagrant VMWare Utility
- Navigate to preview link for Vagrant VMWare Utility install [page](https://dev-portal-git-heat-ff-choreedit-install-landing-json-hashicorp.vercel.app/vagrant/install/vmware)
- There should be no menu items between the OS menu items and Resources

### Packer
- Navigate to preview link for Packer install [page](https://dev-portal-git-heat-ff-choreedit-install-landing-json-hashicorp.vercel.app/packer/install)
- There should be no menu items between Next Steps and Resources

### Consul
- Navigate to preview link for Consul install [page](https://dev-portal-git-heat-ff-choreedit-install-landing-json-hashicorp.vercel.app/consul/install)
- There should be no menu items between Next Steps and Resources

### Terraform
- Navigate to preview link for Terraform install [page](https://dev-portal-git-heat-ff-choreedit-install-landing-json-hashicorp.vercel.app/terraform/install)
- There should be no menu items between Next Steps and Resources

### CODE REVIEW

**Review by pbortnick — APPROVED** [review]
Code changes LGTM 👍

**Review by yhyakuna — APPROVED** [review]

**Comment by heatlikeheatwave** [comment]
cc: @robin-norwood @BrianMMcClain @hashicorp/consul-education 
Just a nudge to get a review from y'all on this PR. Thanks!

**Review by BrianMMcClain — APPROVED** [review]

File: src/content/consul/install-landing.json [hunk 1]
```diff
@@ -13,31 +13,14 @@
 		{
 			"divider": true
 		},
+		{ "heading": "Kubernetes Installs" },
```
Comment (im2nguyen) [inline src/content/consul/install-landing.json:16]:
```suggestion
		{ "heading": "Deploy to Kubernetes" },
```

**Review by im2nguyen — APPROVED** [review]
small nit but looks good to me, thank you!

File: src/content/consul/install-landing.json [hunk 2]
```diff
@@ -13,31 +13,14 @@
...
-			"title": "Consul with Virtual Machines",
-			"fullPath": "/consul/tutorials/get-started-vms/virtual-machine-gs-deploy"
-		},
-		{
-			"title": "HCP Consul",
-			"fullPath": "/consul/tutorials/get-started-hcp/hcp-gs-deploy"
-		},
-		{
-			"divider": true
+			"title": "Consul K8s CLI",
```
Comment (robin-norwood) [inline src/content/consul/install-landing.json:18]:
```suggestion
			"title": "Consul Kubernetes CLI",
```

`K8s` was in the original, but seems to me a bit undiscoverable/informal vs `Kubernetes`.

**Review by robin-norwood — APPROVED** [review]
lgtm!

**Pull request merged**